integration_test/third_party_apps_data/applications/vault/metadata.yaml

```diff
@@ -4,6 +4,117 @@ long_name: Hashicorp Vault
 description: |-
   Vault is an identity-based secrets and encryption management system. This integration Collects Vault's audit logs.
 supported_app_version: ["1.6+"]
+expected_metrics:
+- type: workload.googleapis.com/vault.core.request.count
+  kind: GAUGE
+  monitored_resource: gce_instance
+  value_type: INT64
+  labels: 
+    cluster: .*
+- type: workload.googleapis.com/vault.core.leader.duration
+  optional: true
+  kind: GAUGE
+  monitored_resource: gce_instance
+  value_type: INT64
+  labels: {}
```

Comment:
The scope doc says this needs to have a `cluster` label? Does this of the scope doc need to be updated?

Reply:
Updated the scope doc and it was also a DOUBLE not an INT so updated that as well.